css/index.css

```diff
@@ -1 +1,236 @@
-/* Compile your LESS file! */
\ No newline at end of file
+/* http://meyerweb.com/eric/tools/css/reset/ 
+   v2.0 | 20110126
+   License: none (public domain)
+*/
+html,
+body,
+div,
+span,
+applet,
+object,
+iframe,
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+blockquote,
+pre,
+a,
+abbr,
+acronym,
+address,
+big,
+cite,
+code,
+del,
+dfn,
+em,
+img,
+ins,
+kbd,
+q,
+s,
+samp,
+small,
+strike,
+strong,
+sub,
+sup,
+tt,
+var,
+b,
+u,
+i,
+center,
+dl,
+dt,
+dd,
+ol,
+ul,
+li,
+fieldset,
+form,
+label,
+legend,
+table,
+caption,
+tbody,
+tfoot,
+thead,
+tr,
+th,
+td,
+article,
+aside,
+canvas,
+details,
+embed,
+figure,
+figcaption,
+footer,
+header,
+hgroup,
+menu,
+nav,
+output,
+ruby,
+section,
+summary,
+time,
+mark,
+audio,
+video {
+  margin: 0;
+  padding: 0;
+  border: 0;
+  font-size: 100%;
+  font: inherit;
+  vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article,
+aside,
+details,
+figcaption,
+figure,
+footer,
+header,
+hgroup,
+menu,
+nav,
+section {
+  display: block;
+}
+body {
+  line-height: 1;
+  margin-left: 5%;
+  margin-right: 5%;
+  max-width: 1100px;
+  border: 1px solid black;
+}
+@media (max-width: 500px) {
+  body {
+    width: 80%;
+    background: green;
+  }
+}
+ol,
+ul {
+  list-style: none;
+}
+blockquote,
+q {
+  quotes: none;
+}
+blockquote:before,
+blockquote:after,
+q:before,
+q:after {
+  content: '';
+  content: none;
+}
+table {
+  border-collapse: collapse;
+  border-spacing: 0;
+}
+/* Set every element's box-sizing to border-box */
+* {
+  box-sizing: border-box;
+}
+html,
+body {
+  height: 100%;
+}
+img {
+  width: 45%;
+  height: auto;
+  display: flex;
+  justify-content: flex-start;
+}
+@media (max-width: 500px) {
+  img {
+    justify-content: center;
+    width: 100%;
+  }
+}
+h1 {
+  display: flex;
+  width: 100%;
+  height: 50px;
+  justify-content: center;
+  font-size: 40px;
+}
+p {
+  font-family: "Comic Sans MS";
```

Comment:
1. :thinking: 
2. This won't work for people who don't have Comic Sans installed in their computers (_how dare they!_). To get around this issue, people either send the font along with the CSS, or instruct the browser to download the font from somewhere else (_e.g. Google Fonts_).